components/Apply.js

```diff
@@ -35,6 +35,13 @@ const Apply = () => {
               <Image src="/donate2.png" alt="Donate" width={280} height={30} />
             </a>
           </div>
+          <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=MYCSXB9B4ENP6"
+             target="_blank"
+             rel="noopener noreferrer">
+            <button className="px-10 py-1 block mx-auto bg-red-850 font-black text-lg text-white text-center">
+              Donate Now
+            </button>
+          </a>
```

Comment:
See the above two images. Those two images are also doing the same but in a different way. Those two images are actually anchored tags/links that will eventually redirect you to the donation's website (here Paypal).